yarn-project/pxe/src/pxe_service/pxe_service.ts

```diff
@@ -804,10 +804,6 @@ export class PXEService implements PXE {
     return Promise.resolve(this.synchronizer.getSyncStatus());
   }
 
-  public getKeyStore() {
```

Comment:
Nuked this as it seemed like something we don't want exposed and it was not used.